react/features/toolbox/components/web/Toolbox.js

```diff
@@ -1203,10 +1203,9 @@ class Toolbox extends Component<Props, State> {
         if (this._shouldShowButton('invite')) {
             buttonsRight.push('invite');
         }
-        if (this._shouldShowButton('security') || this._shouldShowButton('info')) {
+        if ((_isModerator && this._shouldShowButton('security')) || this._shouldShowButton('info')) {
```

Comment:
what `info` mean? I think it needs to be
```suggestion
        if ((_isModerator && (this._shouldShowButton('security') || this._shouldShowButton('info'))) {
```

Reply:
The button is showing the current password for the guests (probably "info") and allow changing it for moderators. So I guess the suggested edit makes more sense, since there is already an existing separation of the logic for moderators.

Reply:
I also suppose that because. And depends on that Set we see the buttons.
https://github.com/aeternity/jitsi-meet/blob/cb8183e4821f0b38c56f90307dfb1793ae337cbc/react/features/toolbox/components/web/Toolbox.js#L178-L181

Reply:
@venimus it's confusing because we also have `this._shouldShowButton('invite')` not sure for is `info`. But in UI we got info icon and share tooltip.

Reply:
I added the changes @shapkarin @venimus